### ISSUE TITLE
Plugin can't find modules with Terraform 1.1.x

### ISSUE THREAD

**Issue by gmconte** [VladRassokhin/intellij-hcl#373]
Since I upgraded from Terraform v1.0.10 to v1.1.3 I found that the plugin is unable to find the standard terraform registry modules, hence the autosuggest doesn't work anymore

I believe that this is due to a change in the modules.json generated, causing the plugin to fail the match between the `Source` value in the code with the `Source` value in the json file.

The observed change in modules.json is the following: 

**_modules.json from Terraform v1.0.10_**
```
...
      "Source": "terraform-aws-modules/vpc/aws",
...
```
**_modules.json from Terraform v1.1.3_**
```
...
      "Source": "registry.terraform.io/terraform-aws-modules/vpc/aws",
...
```

As a workaround, I had to modify all my modules to point to the same full path `registry.terraform.io/terraform-aws-modules/xxx` which is not ideal, but at least I can work with it

### Prerequisites
 - [ x ] Ensure you have latest version of plugin installed
 - [ x ] Search for possible issue duplicates

### Installation details
 - [x ] IDE version  **PyCharm 2021.3.1**
 - [ x ] intellij-hcl plugin version **7.14**
 - [x ] Terraform version (`terraform -v`) 
 ```
Terraform v1.1.3
on linux_amd64
+ provider registry.terraform.io/hashicorp/aws v3.72.0
+ provider registry.terraform.io/hashicorp/cloudinit v2.2.0
+ provider registry.terraform.io/hashicorp/external v2.2.0
+ provider registry.terraform.io/hashicorp/helm v2.4.1
+ provider registry.terraform.io/hashicorp/kubernetes v2.7.1
+ provider registry.terraform.io/hashicorp/local v2.1.0
+ provider registry.terraform.io/hashicorp/null v3.1.0
+ provider registry.terraform.io/hashicorp/random v3.1.0
+ provider registry.terraform.io/hashicorp/tls v3.1.0
+ provider registry.terraform.io/terraform-aws-modules/http v2.4.1
```

### Expected Behavior
The plugin doesn't show any warnings

### Actual Behavior
The plugin shows a warning "Cannot locate module locally: Unknown reason" and the auto-complete doesn't work

![image](https://user-images.githubusercontent.com/3528114/149538023-41be7d02-3f1f-43c4-ab7a-a53566077c7c.png)

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. Install terraform v1.1.3
2. run `terraform init -upgrade` to refresh the modules.json file

**Comment by andrewpage** [comment]
Having this same issue.

**Comment by skasukur** [comment]
Same issue with Terraform 1.1.*. I have temporarily reverted to terraform 1.0.11

**Comment by kleijnweb** [comment]
Same here, downgrade works. You can accept the suggested action, it will create `.terraform/modules`, the warning disappears and autocomplete works. But it would be much nicer if the plugin would be just detect `.terraform/modules` in the project root. Now all my submodules have a `.terraform` directory.

**Comment by christian-m** [comment]
The format of the Source attribute for Git-URLs has changed too:

_modules.json from Terraform v0.15_
```
...
      "Source": "git::git@bitbucket.org:sample/url.git?ref=v1.0",
...
```
_modules.json from Terraform v1.1.3_
```
...
      "Source": "git::ssh://git@bitbucket.org/sample/url.git?ref=v1.0",
...
```

**Comment by ohcnkk** [comment]
YouTrack ticket just for reference.

https://youtrack.jetbrains.com/issue/IDEA-287371

Workaround from Jason Capriotti.

https://youtrack.jetbrains.com/issue/IDEA-287371#focus=Comments-27-5743872.0-0

**Comment by ZeroDeth** [comment]
same issue Goland Build #GO-221.5080.224 and Terraform v1.1.8

**Comment by voycey** [comment]
@VladRassokhin - Can this be prioritised please? 4+ months for this is kind of ridiculous

**Comment by mbrancato** [comment]
From the example above, this works for me as workaround - it seems Terraform is being explicit about the registry source now when its implied in the HCL. 

```hcl
module "my_module" {
  source      = "terraform-aws-modules/vpc/aws"
}
```

works fine when set as

```hcl
module "my_module" {
  source      = "registry.terraform.io/terraform-aws-modules/vpc/aws"
}
```

**Comment by jkroepke** [comment]
Seems fixed in latest version

**Comment by ohcnkk** [comment]
Yes, this issue is fixed in latest IDE versions. Please refer https://youtrack.jetbrains.com/issue/IDEA-287371.
@gmconte May be this issue can be closed.

**Comment by markmandel** [comment]
I'm still seeing this issue with Terraform 1.3.7
